test/conformance.go

```diff
@@ -38,20 +38,22 @@ const (
 	Autoscale           = "autoscale"
 	EmptyDir            = "emptydir"
 	Failing             = "failing"
+	GRPCPing            = "grpc-ping"
+	HTTPProxy           = "httpproxy"
+	HelloHTTP2          = "hellohttp2"
 	HelloVolume         = "hellovolume"
 	HelloWorld          = "helloworld"
-	HTTPProxy           = "httpproxy"
```

Comment:
HTTPProxy should come after the Hello ones

Reply:
oh, good catch, apparently `sort` sorts capital letters first by default in whatever locale Im in